README.md

```diff
@@ -8,51 +8,22 @@
 [![Swagger](https://img.shields.io/badge/-Swagger-%23Clojure?style=for-the-badge&logo=swagger&logoColor=white)](https://funding-service-design-account-store-dev.london.cloudapps.digital/#/default/core.account.post_account_by_email)
 ![Gunicorn](https://img.shields.io/badge/gunicorn-%298729.svg?style=for-the-badge&logo=gunicorn&logoColor=white)
 
-Repo for the funding service design account store.
+This repository is designed to handle the generating and providing of JSON payloads for accounts. It takes an email address and creates/returns a corresponding payload of account json
 
-Built with Flask + Connexion.
+[Developer setup guide](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-setup.md)
 
-Takes a email address and creates/returns a corresponding payload of account json.
 
-## Prerequisites
-- python ^= 3.10
+This service depends on:
+- A postgres database
+- funding-service-design-application-store
+- funding-service-design-assessment-store
 
-# Getting started
+# IDE Setup
+[Python IDE Setup](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-ide-setup.md)
 
-## Installation
-
-Clone the repository
-
-### Create a Virtual environment
-
-    python3 -m venv .venv
-
-### Enter the virtual environment
-
-...either macOS using bash:
-
-    source .venv/bin/activate
-
-...or if on Windows using Command Prompt:
-
-    .venv\Scripts\activate.bat
-
-### Install dependencies
-From the top-level directory enter the command to install pip and the dependencies of the project
-
-    python3 -m pip install --upgrade pip && pip install -r requirements-dev.txt
-
-### Install pre-commit hooks
-    pre-commit install
-
-### Updating dependencies
-requirements-dev.txt and requirements.txt are updated using [pip-tools pip-compile](https://github.com/jazzband/pip-tools)
-To update requirements please manually add the dependencies in the .in files (not the requirements.txt files)
-Then run (in the following order):
-
-    pip-compile requirements.in
-
-    pip-compile requirements-dev.in
+# Data
+## Local DB Setup
+General instructions for local db development are available here: [Local database development](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-db-development.md)
 
```

Comment:
Most of this DB stuff below is covered in the shared readme, please reword and remove the stuff that's covered elsewhere.

The env config stuff is useful tho and should be moved to one of the shared readmes as it applies to all repos